src/components/organisms/AdminVenueView/components/Timing/Timing.tsx

```diff
@@ -32,30 +34,36 @@ export const Timing: React.FC<TimingProps> = ({
       <div className="Timing__left">
         <div className="Timing__left-bottombar">
           <div className="Timing__left-bottombar-btnleft">
-            <button
-              // href="/admin-ng"
-              className="Timing__left-bottombar-btngrey Timing__btn"
-            >
-              <FontAwesomeIcon
-                icon={faHome}
-                className="edit-button__icon"
-                size="lg"
-              />
-            </button>
+            {onClickHome && (
```

Comment:
Rendering content depending on callback prop seems weird, can we just change `onClickHome` to be required prop instead? WDYT?

Reply:
The reason is to have same props for all 3 tabs i.e. the components being displayed, that way any one of them may or may not have a home/back/next buttons, but they will not have a reference if they're first or last in the flow. The concern of the flow will rest in `Spaces` component and how it arranges them.